package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@wazo/sdk",
-  "version": "0.42.0",
+  "version": "0.43.0-rc.0",
```

```suggestion
  "version": "0.42.0",
```
should be done from master instead of PR ?